platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/Transform.java

```diff
@@ -174,27 +174,23 @@ private long getDurationNano(long durationMs) {
     return cameraPosition.zoom;
   }
 
-  void zoom(boolean zoomIn) {
-    zoom(zoomIn, -1.0f, -1.0f);
-  }
-
-  void zoom(boolean zoomIn, float x, float y) {
+  void zoom(boolean zoomIn, float focalX, float focalY) {
```

Comment:
We should start considering refactoring [data clumps](https://martinfowler.com/bliki/DataClump.html) like `focalX`/ `focalY`. Here we could [introduce a parameter object](https://refactoring.com/catalog/introduceParameterObject.html).